src/Kdyby/Console/DI/ConsoleExtension.php

```diff
@@ -82,18 +82,16 @@ public function loadConfiguration()
 		Nette\Utils\Validators::assert($config, 'array');
 		foreach ($config['commands'] as $i => $command) {
 			$def = $builder->addDefinition($this->prefix('command.' . $i));
-			list($def->factory) = Nette\DI\Compiler::filterArguments(array(
-				is_string($command) ? new Statement($command) : $command
-			));
-
-			if (class_exists($def->factory->entity)) {
-				$def->class = $def->factory->entity;
-			}
-
-			$def->setAutowired(FALSE);
-			$def->setInject(FALSE);
```

Comment:
I need the disabled inject here, it's an optimalization - it's meant to allow you get your sevices using property injection which is called in runtime `Kdyby\Console\Application::doRunCommand()` so you don't have to initialize the dependency graph of all commands which are not gonna be run anyway.